data/json/mapgen/lab/lab_central.json

```diff
@@ -5,56 +5,35 @@
     "om_terrain": [ "central_lab_entrance" ],
     "weight": 100,
     "object": {
-      "fill_ter": "t_dirt",
+      "predecessor_mapgen": "forest",
       "rows": [
-        ",,,,,,,,,,,,,,,,,,,,,,,,",
-        ",,,,,,,,,,,,,,,,,,,,,,,,",
-        ",,,,                ,,,,",
-        ",,,                  ,,,",
-        ",,                    ,,",
-        ",,                    ,,",
-        ",,                    ,,",
-        ",,                    ,,",
-        ",,                    ,,",
-        ",,                    ,,",
-        ",          7           ,",
-        ",         7o7          ,",
-        ",          7           ,",
-        ",,                    ,,",
-        ",,                    ,,",
-        ",,                    ,,",
-        ",,                    ,,",
-        ",,                    ,,",
-        ",,                    ,,",
-        ",,                    ,,",
-        ",,,                  ,,,",
-        ",,,,                ,,,,",
-        ",,,,,,,,,,,,,,,,,,,,,,,,",
-        ",,,,,,,,,,,,,,,,,,,,,,,,"
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "          ...           ",
+        "          .o.           ",
+        "          ...           ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        "
       ],
-      "terrain": {
-        " ": [ "t_dirt", "t_grass", "t_grass", "t_grass", "t_grass", "t_grass", "t_grass", "t_underbrush", "t_tree_young", "t_tree" ],
-        ",": [
-          "t_dirt",
-          "t_dirt",
-          "t_grass",
-          "t_grass",
-          "t_grass",
-          "t_grass",
-          "t_grass",
-          "t_grass",
-          "t_grass",
-          "t_grass",
-          "t_grass",
-          "t_grass",
-          "t_grass",
-          "t_underbrush",
-          "t_tree_young",
-          "t_tree"
-        ],
-        "o": "t_manhole_cover",
-        "7": "t_tree"
-      }
+      "flags": [ "ERASE_ALL_BEFORE_PLACING_TERRAIN" ],
+      "terrain": { ".": "t_region_groundcover_barren", "o": "t_manhole_cover" }
```

Comment:
This looks like it's going to make these manholes way too easy to find now. They should probably keep the trees around it as before in order to keep central labs secret.

Reply:
They have always been easy to find if you knew what to look out for.  One reason I made this PR was to make the forest tile where the manhole is and the surrounding woods indistinguishable from each other. I made a compromise by removing the trees around the manhole.  Not to mention blocking your only emergency exit with oak trees is big safety hazard.

Reply:
Seems reasonable enough. Awesome changes by the way, can't wait to see them in game.